src/Symfony/Component/Debug/Exception/FlattenException.php

```diff
@@ -34,15 +34,20 @@ class FlattenException
     private $line;
 
     public static function create(\Exception $exception, $statusCode = null, array $headers = array())
+    {
+        return static::createFromThrowable($exception, $statusCode, $headers);
+    }
+
+    public static function createFromThrowable(\Throwable $throwable, ?int $statusCode = null, array $headers = array()): self
     {
         $e = new static();
-        $e->setMessage($exception->getMessage());
```

Comment:
sorry, anynother change: let's keep the old name, it's still fine

Reply:
Fair enough.